### ISSUE TITLE
Upgrade 'Data Flow' to 1.5.0.RELEASE

### ISSUE THREAD

**Issue by andrewtyt** [donovanmuller/spring-cloud-dataflow-server-openshift#33]
Hello Donovan, 

Are there any plans to upgrade 'Spring Cloud Data Flow' to [1.5.0.RELEASE](https://spring.io/blog/2018/05/16/spring-cloud-data-flow-1-5-0-released) version? 

I've managed to built&run docker image with the following changes:

spring-cloud-dataflow-server-openshift [pom.xml](https://github.com/donovanmuller/spring-cloud-dataflow-server-openshift/blob/master/pom.xml):
```
<parent>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-dataflow-parent</artifactId>
    <version>1.5.0.RELEASE</version>
    <relativePath />
</parent>
...
<properties>
    <spring-cloud-deployer-openshift.version>1.3.2.RELEASE</spring-cloud-deployer-openshift.version>
    <spring-cloud-dataflow.version>1.5.0.RELEASE</spring-cloud-dataflow.version>
    <spring-cloud-deployer-spi.version>1.3.2.RELEASE</spring-cloud-deployer-spi.version>
</properties>
```
spring-cloud-deployer-openshift [pom.xml](https://github.com/donovanmuller/spring-cloud-deployer-openshift/blob/master/pom.xml): 
```
<parent>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-deployer-parent</artifactId>
    <version>1.3.2.RELEASE</version>
</parent>
...
<properties>
    <spring-cloud-deployer-kubernetes.version>1.3.4.RELEASE</spring-cloud-deployer-kubernetes.version>
    <spring-cloud-deployer.version>1.3.2.RELEASE</spring-cloud-deployer.version>
    <spring-cloud.version>Edgware.SR3</spring-cloud.version>
</properties>
```

**Comment by donovanmuller** [comment]
Hi @andrewtyt,

Unfortunately I have had very little time over the past few months to dedicate to maintaining this project 😔

To upgrade this project to the latest Spring Cloud Data Flow release, you would have to:

* Upgrade the [`spring-cloud-deployer-openshift`](https://github.com/donovanmuller/spring-cloud-deployer-openshift) project (where the actual deployer lives) to the equivalent versions as per below
* `spring-cloud-dataflow-parent` to `1.5.0.RELEASE`
* `spring-cloud-deployer-kubernetes` to [`1.3.4.RELEASE`](https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/releases/tag/v1.3.4.RELEASE)

as well as make sure the new Skipper deployments work as expected.

I can't make any promises but I'll try my best to update both this and `spring-cloud-deployer-openshift` to the latest release this week.

**Comment by andrewtyt** [comment]
In case it’s helpful here are git [patches.zip](https://github.com/donovanmuller/spring-cloud-dataflow-server-openshift/files/2044754/patches.zip) with changes I've done to my local repo. 
Thank you!

**Comment by gdubya** [comment]
Also interested in an updated release :)

**Comment by qnob** [comment]
Hi all,
are any news about the upgrade to the current version? Had anyone experience with the integration of the latest releases?

We are looking forward to thr new dashboard UI and scheduler feature.

Thanks for any reply.
Kuno

**Comment by donovanmuller** [comment]
@qnob Unfortunately I don't have any free time to maintain this project currently. Therefore I cannot promise there will be any new releases.

I believe @gdubya might be working on some updates. Perhaps he might be able to provide some feedback on a new release if at all possible?

**Comment by gdubya** [comment]
Yeah, I'm interested in getting it working with the upcoming Spring Cloud Greenwich release so I'll spend a few hours to try and get that working, but I can't guarantee anything.

@donovanmuller will you keep this repo alive since it's referenced from the SCDF homepage?

**Comment by gdubya** [comment]
That's a good point, which I hadn't really considered. I'm pretty confident that we don't actually need any of the remaining Openshift-specific features at the moment.

Our openshift cluster is _slightly_ behind latest (openshift v3.9, kubernetes v1.9.1), but maybe that won't matter. I'll test the latest kubernetes deployer and report back.

**Comment by qnob** [comment]
Hi guys,
thank you a lot for your time. Just right now we need to focus on customer requirements :), however, I'll try my best to support you guys.
Kuno

**Comment by donovanmuller** [comment]
@gdubya curious as to whether SCDF Kubernetes server deployed successfully to your OpenShift cluster (assuming you have tried that)?

**Comment by gdubya** [comment]
@donovanmuller Basically yes, it works. But i'm still figuring out how it will work for us to deploy tasks and apps. SDCF in kubernetes must pull apps from a docker repo (default dockerhub), but our cluster firewall prevents downloading from external sites by default. So i need to figure out how to install an app into SCDF from the internal openshift registry (docker-registry.default.svc:5000). The documentation talks about configuring a secret for this (http://docs.spring.io/spring-cloud-dataflow/docs/2.0.0.BUILD-SNAPSHOT/reference/htmlsingle/#_private_docker_registry) but that shouldn't be necessary for the integrated registry, should it?
